content/en/docs/userguide/try_request_routing.md

```diff
@@ -0,0 +1,173 @@
+---
+draft: false
+linktitle: Try Request Routing
+menu:
+  docs:
+    parent: user guide
+    weight: 5
+title: Try Request Routing
+toc: true
+type: docs
+
+
+---
+
+### Preparation
+
+- install Kmesh, refer [install-kmesh | Quick Start | Kmesh](https://kmesh.net/en/docs/setup/quickstart/#install-kmesh)
+
+Refer [Install Waypoint | Kmesh](https://kmesh.net/en/docs/userguide/install_waypoint/#preparation) in following steps
+
+- Deploy sample application
+- Deploy bookinfo
+- Deploy sleep as curl client
+- Install service granulairty waypoint
```

Comment:
- Make default namespace managed by Kmesh
- Deploy bookinfo as sample application and sleep as cur client
- Install service granularity waypoint for reviews service